Makefile

```diff
@@ -224,8 +224,13 @@ kind-deploy-controller: manifests
 	-kubectl create ns $(KIND_NAMESPACE)
 	kubectl apply -f deploy/operator.yaml -n $(KIND_NAMESPACE)
 
+HOSTED ?= none
+
 .PHONY: kind-deploy-controller-dev
-kind-deploy-controller-dev: kind-deploy-controller
+kind-deploy-controller-dev:
+	if [ "$(HOSTED)" != "hosted" ]; then\
+		$(MAKE) kind-deploy-controller;\
```

Comment:
In our hosted mode, the addon-controller deploys propagator-controller in advance, so kind-deploy-controller isn't needed

Reply:
I can close this pr Thanks!